apps/next_dev_portfolio/components/Hero.js

```diff
@@ -0,0 +1,50 @@
+import Image from "next/image";
+import Button from "./Button";
+import { DotSmall } from "./Dot";
+
+export default function Hero() {
+  const headers = ["Lorem Ipsum", "Lorem Prosum"];
```

Comment:
Things that are not changing or not derived from state/props can be placed outside the component.